scripts/orthophotomap_to_patches_dataset_multiprocess.py

```diff
@@ -9,7 +9,11 @@
 import skimage as ski
 import skimage.io
 import os
+import multiprocessing
```

Comment:
ponownie:w międzyczasie na innym branchu został dokończony skrypt generujący dataset. Co prawda tylko jednowątkowo, ale na razie to nie przeszkadza ( bo i tak puszczając generowanie na kilku zbiorach jednocześnie daje efekt "zrównoleglenia" :P)